apps/cowswap-frontend/src/modules/tenderly/hooks/useTenderlyBundleSimulation.ts

```diff
@@ -0,0 +1,106 @@
+import { useCallback } from 'react'
+
+import { useWalletInfo } from '@cowprotocol/wallet'
+
+import useSWR from 'swr'
+
+import { useHooks } from 'modules/hooksStore'
+import { useOrderParams } from 'modules/hooksStore/hooks/useOrderParams'
+
+import { useTokenContract } from 'common/hooks/useContract'
+
+import { useGetTopTokenHolders } from './useGetTopTokenHolders'
+
+import { completeBundleSimulation, preHooksBundleSimulation } from '../utils/bundleSimulation'
+import { generateNewSimulationData, generateSimulationDataToError } from '../utils/generateSimulationData'
+import { getTokenTransferInfo } from '../utils/getTokenTransferInfo'
+
+export function useTenderlyBundleSimulate() {
```

Comment:
The fn name still doesn't match 😬 

`useTenderlyBundleSimulation.ts` vs `useTenderlyBundleSimulate`